### PR TITLE
Update index.md | top and left is supported in Firefox >= 109

### LINKED ISSUE #36509: `top` and `left` is supported in Firefox >= 109
### MDN URL

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/windows/create

### What specific section or headline is this issue about?

parameters

### What information was incorrect, unhelpful, or incomplete?

The description of `top` and `left` says `In Firefox, this value currently is ignored for popups (bug 1271047) but can be set using browser.windows.update().`

### What did you expect to see?

The issue should be applied only for Firefox < 109.

### Do you have any supporting links, references, or citations?

[Bug 1271047](https://bugzilla.mozilla.org/show_bug.cgi?id=1271047)

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/mozilla/add-ons/webextensions/api/windows/create`
* MDN URL: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/windows/create
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/mozilla/add-ons/webextensions/api/windows/create/index.md
* Last commit: https://github.com/mdn/content/commit/b8a0743ca8b1e1b1b1a95cc93a4413c020f11262
* Document last modified: 2024-09-01T17:04:58.000Z

</details>

### PR DESCRIPTION
**Pull request by Piyush-r-bhaskar**
### Description

Top and left is supported in Firefox >= 109

### Additional details

https://bugzilla.mozilla.org/show_bug.cgi?id=1271047
FIX - https://phabricator.services.mozilla.com/D73419?id=649918

### Related issues and pull requests

🔨 If this fully resolves a GitHub issue, use "Fixes #36509"
👉 Highlight related pull requests using "Relates to #36509"
❗ If another pull request should be merged first, use "**Depends on:** https://github.com/mdn/browser-compat-data/pull/24850"

### CODE REVIEW

File: files/en-us/mozilla/add-ons/webextensions/api/windows/create/index.md [hunk 1]
```diff
@@ -49,15 +49,15 @@ let creating = browser.windows.create(
     - `incognito` {{optional_inline}}
       - : `boolean`. Whether the new window should be an incognito (private) window. Note that if you specify `incognito` and `tabId`, the ID must refer to a private tab — that is, you can't move a non-private tab to a private window.
     - `left` {{optional_inline}}
-      - : `integer`. The number of pixels to position the new window from the left edge of the screen. If not specified, the new window is offset naturally from the last focused window. This value is ignored for panels. (In Firefox, this value currently is ignored for popups (bug 1271047) but can be set using browser.windows.update().)
+      - : `integer`. The number of pixels to position the new window from the left edge of the screen. If not specified, the new window is offset naturally from the last focused window. (For Firefox versions earlier than 109, `left: value` is ignored for popups (bug 1271047) and workaround is using `browser.windows.update()`. From Firefox Version 109, the `left: value` is considered (fix D73419).)
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/windows/create/index.md:52]:
```suggestion
      - : `integer`. The number of pixels to position the new window from the left edge of the screen. If not specified, the new window is offset naturally from the last focused window. (Was not supported for popups in Firefox 108 or earlier, where positioning the window using {{WebExtAPIRef("windows.update()")}} could be used as a workaround.)
```

File: files/en-us/mozilla/add-ons/webextensions/api/windows/create/index.md [hunk 2]
```diff
@@ -49,15 +49,15 @@ let creating = browser.windows.create(
...
     - `tabId` {{optional_inline}}
       - : `integer`. If included, moves a tab of the specified ID from an existing window into the new window.
     - `titlePreface` {{optional_inline}}
       - : `string`. Use this to add a string to the beginning of the browser window's title. Depending on the underlying operating system, this might not work on browser windows that don't have a title (such as about:blank in Firefox).
     - `top` {{optional_inline}}
-      - : `integer`. The number of pixels to position the new window from the top edge of the screen. If not specified, the new window is offset naturally from the last focused window. This value is ignored for panels. (In Firefox, this value currently is ignored for popups (bug 1271047) but can be set using browser.windows.update().)
+      - : `integer`. The number of pixels to position the new window from the top edge of the screen. If not specified, the new window is offset naturally from the last focused window. (For Firefox versions earlier than 109, `top: value` is ignored for popups (bug 1271047) and workaround is using `browser.windows.update()`. From Firefox version 109, the `top: value` is considered (fix D73419).)
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/windows/create/index.md:60]:
```suggestion
      - : `integer`. The number of pixels to position the new window from the top edge of the screen. If not specified, the new window is offset naturally from the last focused window. (Was not supported for popups in Firefox 108 or earlier, where positioning the window using {{WebExtAPIRef("windows.update()")}} could be used as a workaround.)
```

**Review by rebloor — CHANGES_REQUESTED** [review]
@Piyush-r-bhaskar thanks for suggesting these changes. I've suggested some updates. In addition, the information about the fix should be included in the 109 release note.

**Comment by Piyush-r-bhaskar** [comment]
Thanks @rebloor for the suggestions.

**Comment by rebloor** [comment]
@Piyush-r-bhaskar you will find the release note file that needs updating at /files/en-us/mozilla/firefox/releases/109/index.md

**Comment by Piyush-r-bhaskar** [comment]
Thank You @rebloor 

I have updated & opened PR https://github.com/mdn/content/pull/36588

**Review by rebloor — APPROVED** [review]

**Comment by rebloor** [comment]
Thanks @Piyush-r-bhaskar

**Comment by Piyush-r-bhaskar** [comment]
Thanks so much for merging the PR @rebloor and for the support during the process! I'm glad to have contributed, and I'm looking forward to working on more documentation or other projects in the future.

**Pull request merged**